Tests/AppTests/__Snapshots__/WebpageSnapshotTests/test_PackageShowView.1.txt

```diff
@@ -121,7 +121,7 @@
               </ul>
             </li>
             <li>
-              <p>Version 
+              <p>Versions 
```

Comment:
This is a genuine fix in the markup, we were listing this as `Version X and Y` before

Reply:
Actually, this is generated from `listPhrase` so confirming there's no comma I believe. (see comment below)